nix/ext/hypopg.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "hypopg";
+  version = "1.3.1";
```

Comment:
any way we can connect this up to values in ansible var file here https://github.com/supabase/postgres/blob/develop/ansible/vars.yml ?

it doesn't necessarily need to be in this PR but it would be great to make everything updatable from a single location

Reply:
@olirice  I am going to look into consuming that file, possibly matching on the extname, then using the version, etc, and so then being able to do the automation you describe.  I'll follow up with another PR that focuses just on that bit. Thanks for this suggestion, it's a great solution to integration with deployment.

Reply:
Will resolve with the plan to create an implementation above